src/components/AddProjectForm.js

```diff
@@ -118,7 +118,8 @@ const AddProjectForm = (props) => {
         }
         if (projectToggl) {
             if (!verifyTogglURL(projectToggl)) {
-                setCreateProjectError('The Toggl URL is invalid')
+                setCreateProjectError('The Toggl URL is invalid.')
+                setCreateProjectError('Please follow the following format https://track.toggl.com/###/projects/###')
```

Comment:
Let's match this with https://github.com/setlife-network/trinary/pull/571/files#diff-ebbc47c19924a093eb3b1dc5e7b0f9948b71855eb4f834a4ec691758a6cd3508R102 by creating a constant `INVALID_TOGGL_URL_ERROR_MESSAGE` and importing it in both files